packages/langchain/src/orchestration/client.test.ts

```diff
@@ -129,4 +158,93 @@ describe('orchestration service client', () => {
       'Request failed with status code 400'
     );
   }, 1000);
+
+  it('supports streaming responses', async () => {
+    mockStreamInferenceWithResilience();
+
+    const client = new OrchestrationClient(config);
+    const stream = await client.stream([]);
+    let iterations = 0;
+    const maxIterations = 2;
+    let intermediateChunk: AIMessageChunk | undefined;
+
+    for await (const chunk of stream) {
+      iterations++;
+      intermediateChunk = !intermediateChunk
+        ? chunk
+        : intermediateChunk.concat(chunk);
+      if (iterations >= maxIterations) {
+        break;
+      }
+    }
+    expect(intermediateChunk).toBeDefined();
+    expect(intermediateChunk!.content).toBeDefined();
+    expect(intermediateChunk!.content).toEqual(
+      'The SAP Cloud SDK is a comprehensive development toolkit designed to simplify and accelerate the cre'
+    );
+  });
+
+  it('streaming with abort signal', async () => {
+    mockStreamInferenceWithResilience();
+    const client = new OrchestrationClient(config);
+    const controller = new AbortController();
+    const { signal } = controller;
+    const stream = await client.stream([], { signal });
+    const streamPromise = async () => {
+      for await (const _chunk of stream) {
+        controller.abort();
+      }
+    };
+
+    await expect(streamPromise()).rejects.toThrow();
```

Comment:
[pp] Check also the throw content.
[pp] Rename the `streamPromise`, it is a function not a promise.

Reply:
The checked throw content is different, not the error I initially threw( this was unreachable code, so I removed it).
Also did the renaming.